src/extension/background-script/actions/accounts/export.ts

```diff
@@ -0,0 +1,26 @@
+import { decryptData } from "~/common/lib/crypto";
+import state from "~/extension/background-script/state";
+import type { MessageAccountEdit } from "~/types";
+
+const exportAccount = async (message: MessageAccountEdit) => {
+  const accounts = state.getState().accounts;
+  const password = state.getState().password as string;
```

Comment:
We should try to avoid `ad`. It's not good practice.

Reply:
You mean `as`? It is causing a typescript error below where we use it in the `decryptData` function...

Reply:
Arrr, yeah, `as`... pity, ok